actions/compute-artifact-description/main.go

```diff
@@ -93,7 +93,9 @@ func main() {
 	}
 	sort.Strings(deps)
 
-	fmt.Printf("::set-output name=artifact-reference-description::%s", strings.Join(deps, "%0A")) // linefeed
+	actions.Outputs{
+		"artifact-reference-description": strings.Join(deps, "%0A"), // linefeed
+	}.Write()
```

Comment:
Is it still necessary to join on `%0A` here (urlencoded `\n`)? If we are handling multiline strings when we write them out?